docs/pages/deploy-a-cluster/high-availability.mdx

```diff
@@ -0,0 +1,369 @@
+your Teleport Proxy Service.
+
+For high-availability deployments, however, it is much more efficient to use a
+separate system to request TLS credentials from Let's Encrypt and provision
+Teleport instances with them. In this case, with multiple Teleport instances
+behind a load balancer, you will need to use the [ACME
+DNS-01](https://letsencrypt.org/docs/challenge-types/#dns-01-challenge)
+challenge to demonstrate domain name ownership to Let's Encrypt. In this
+challenge, your TLS credential provisioning system creates a DNS TXT record with
+a value expected by Let's Encrypt.
+
+In the configuration we are demonstrating in this guide, each Teleport instance
+expects TLS credentials for HTTPS to be available at the file paths
+`/etc/teleport-tls/tls.key` (private key) and `/etc/teleport-tls/tls.crt`
+(certificate).
+
+## DNS service
+
+Set up a DNS zone where you can create records for Teleport, e.g., an Amazon
+Route 53 hosted zone or Google Cloud DNS zone.
+
+You must create a DNS A record that points to the address of your load balancer. 
+
+(!docs/pages/includes/dns-app-access.mdx!)
+
+<Admonition title="Required permissions">
+
+The TLS credential provisioner we mentioned earlier needs permissions to manage
+DNS records in order to satisfy Let's Encrypt's DNS-01 challenge. If you are
+using cloud-managed solutions, you should use your cloud provider's RBAC system
+(e.g., AWS IAM) to grant a role to the Proxy Service to manage DNS records. 
+
+</Admonition>
+
+## Teleport instances
+
+Run the Teleport Auth Service and Proxy Service as a scalable group of compute
+resources, for example, a Kubernetes `Deployment`  or AWS Auto Scaling group.
+This requires running the `teleport` binary on each Kubernetes pod or virtual
+machine or in your group. In the [Configuration](#configuration) section, we
+will show you how to configure each binary for high availability.
+
+### Open ports
+
+Ensure that, on each Teleport instance, the following ports allow traffic from
+the load balancer. If you do not plan to run a service associated with a given
+port, you can plan to leave that port closed.
+
+| Port | Description |
+| - | - |
+| `3021` | Port used by Teleport Proxy Service instances to dial agents in [Proxy Peering mode](../management/operations/proxy-peering.mdx). |
+| `3022` | SSH port. This is Teleport's equivalent of port `22` for SSH. |
+| `3023` | SSH port for clients connect to. The Proxy Service will forward this connection to port `3022` on the destination Node. |
+| `3024` | SSH port used to create reverse SSH tunnels from behind-firewall environments. |
+| `3025` | TLS port used by the Auth Service to serve its gRPC API to Teleport services  in a cluster. |
+| `443` | HTTPS connections to authenticate `tsh` users into the cluster. The same connection is used to serve a Web UI. |
+| `3026` | HTTPS Kubernetes proxy |
+| `3027` | Kubernetes Service |
+| `3028` | Desktop Service |
```

Comment:
I'm not sure if this is the same case as 3022 and 3027. It sounds like.

Reply:
Yes, I believe so; `windows_desktop_service` was pretty much initially a copy and paste with `s/kubernetes_service/windows_desktop_service/g` and `s/3027/3028/g`